kibana-reports/server/routes/utils/__tests__/savedSearchReportHelper.test.ts

```diff
@@ -22,7 +22,7 @@ import { reportSchema } from '../../../model';
  */
 const input = {
   query_url:
-    'http://localhost:5601/app/kibana#/search/7adfa750-4c81-11e8-b3d7-01146121b73d',
+    'http://localhost:5601/app/discover#/view/7adfa750-4c81-11e8-b3d7-01146121b73d',
```

Comment:
Does this PR include a version change? Why are we changing the URL?

Reply:
That was a typo, it should be discover view. But anyway, this field doesn't affect the test